src/GafferSceneUI/SceneView.cpp

```diff
@@ -1795,6 +1851,17 @@ bool SceneView::keyPress( GafferUI::GadgetPtr gadget, const GafferUI::KeyEvent &
 		if( !b.isEmpty() && viewportGadget()->getCameraEditable() )
 		{
 			viewportGadget()->frame( b );
+
+			if( m_camera->lookThroughCameraIsDistantLight() )
+			{
+				FloatPlug* distantAperture = m_camera->plug()->getChild<FloatPlug>( "lightLookThroughDistantAperture" );
+				float bSize = std::max( b.size()[0], std::max( b.size()[1], b.size()[2] ) );
+				if( bSize > distantAperture->getValue() )
+				{
+					distantAperture->setValue( bSize );
+				}
+			}
+
 			if( event.modifiers == KeyEvent::Control )
```

Comment:
Forgot to mention in comment - this branch of the original code never triggers, because of the above test that `modifiers == None`?  Not sure how useful it is, maybe it should be removed?

Reply:
Opened https://github.com/GafferHQ/gaffer/pull/4428 to take care of this.